website/content/en/docs/building-operators/golang/references/env-test-setup.md

```diff
@@ -0,0 +1,64 @@
+---
+title: EnvTest Setup
+linkTitle: EnvTest Setup
+weight: 50
+---
+
+## Overview 
+
+This document describes how to configure the environment for the [controller tests][controller-test] which uses [envtest][envtest] and is supported by the SDK. 
+
+## Installing prerequisites
+
+[Envtest][envtest] requires that `kubectl`, `api-server` and `etcd` be present locally. You can use this [script][script] to download these binaries into the `testbin/` directory. It may be convenient to add this script your Makefile as follows:
+
+```sh
+# Setup binaries required to run the tests
+# See that it expects the Kubernetes and ETCD version
+K8S_VERSION = v1.18.2
+ETCD_VERSION = v3.4.3
+testbin:
+	curl -sSLo setup_envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/kubebuilder/master/scripts/setup_envtest_bins.sh 
+	chmod +x setup_envtest.sh
+	./setup_envtest.sh $(K8S_VERSION) $(ETCD_VERSION)
+```
+
+
+The above script sets these environment variables to specify where test binaries can be found. Then, see that if you would like to NOT use the script you can use the same configuration by informing the path of your binaries: 
```

Comment:
```suggestion
The above script sets these environment variables to specify where test binaries can be found. In case you would like to not use the script then, is possible to do the same configuration to inform the path of your binaries: 
```